apps/language_server/lib/language_server/providers/execute_command/manipulate_pipes/ast.ex

```diff
@@ -64,4 +64,17 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.AST d
   defp do_to_pipe(node, acc) do
     {node, acc}
   end
+
+  defp fix_escape_chars(parsed, original) do
```

Comment:
This fix doesn't work properly because sigil delimiters changed to `"` prior to 1.10

Some parsing of the AST will be needed to treat sigils